### PR TITLE
Added tooltip for uptime.

### PR DESCRIPTION
**Pull request by hthayko**

### CODE REVIEW

File: views/home.html [hunk 1]
```diff
@@ -48,7 +48,14 @@
         <th class='version'>Version</th>
         <th>Inbound Connections</th>
         <th>Outbound Connections</th>
-        <th>Uptime</th>
+        <th class="uptime">
+          Uptime
+          <button type="button" class="tooltip-btn btn btn-default"
+            data-placement="top"
+            title="uptime of peer's longest current connection"
```
Comment (wilsonianb) [inline views/home.html]: How about `age of peer's longest current connection`?

Comment (hthayko) [inline views/home.html]: you don't like the work 'uptime' ? :))

Comment (wilsonianb) [inline views/home.html]: It's just weird using "uptime" in our definition of "uptime"

Comment (hthayko) [inline views/home.html]: that's why I wanted to change the column altogether. uptime of the smth is already defined as up-time, i.e. the time smth is up. we need a different word here

Comment (hthayko) [inline views/home.html]: okay

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**